std_msgs/Header header
geometry_msgs/PoseStamped pose_stamped
geometry_msgs/TwistStamped twist_stamped
geometry_msgs/WrenchStamped wrench_stamped
std_msgs/Int32 grey_button
std_msgs/Int32 white_button
